examples/audio-classification/README.md

```diff
@@ -0,0 +1,171 @@
+
+The following examples showcase how to fine-tune `Wav2Vec2` for audio classification on Habana Gaudi.
+
+Speech recognition models that have been pretrained in unsupervised fashion on audio data alone, *e.g.* [Wav2Vec2](https://huggingface.co/transformers/main/model_doc/wav2vec2.html), have shown to require only very little annotated data to yield good performance on speech classification datasets.
+
+## Single-HPU
+
+The following command shows how to fine-tune [wav2vec2-base](https://huggingface.co/facebook/wav2vec2-base) on the 🗣️ [Keyword Spotting subset](https://huggingface.co/datasets/superb#ks) of the SUPERB dataset on a single HPU.
+
+```bash
+python run_audio_classification.py \
+    --model_name_or_path facebook/wav2vec2-base \
+    --dataset_name superb \
+    --dataset_config_name ks \
+    --output_dir /tmp/wav2vec2-base-ft-keyword-spotting \
+    --overwrite_output_dir \
+    --remove_unused_columns False \
+    --do_train \
+    --do_eval \
+    --learning_rate 3e-5 \
+    --max_length_seconds 1 \
+    --attention_mask False \
+    --warmup_ratio 0.1 \
+    --num_train_epochs 5 \
+    --per_device_train_batch_size 256 \
+    --per_device_eval_batch_size 256 \
+    --dataloader_num_workers 4 \
+    --seed 27 \
+    --use_habana \
+    --use_lazy_mode \
+    --gaudi_config_name Habana/wav2vec2 \
+    --throughput_warmup_steps 2
+```
+
+On a single HPU, this script should run in ~13 minutes and yield accuracy of **97.96%**.
+
+> If your model classification head dimensions do not fit the number of labels in the dataset, you can specify `--ignore_mismatched_sizes` to adapt it.
+
+> If your data has a sampling rate which is different from the one of the data the model was trained on, this script will raise an error.
+> Resampling with the `datasets` library is not supported on HPUs yet.
+
+
+## Multi-HPU
+
+The following command shows how to fine-tune [wav2vec2-base](https://huggingface.co/facebook/wav2vec2-base) for 🌎 **Language Identification** on the [CommonLanguage dataset](https://huggingface.co/datasets/anton-l/common_language) on 8 HPUs.
+
+```bash
+python ../gaudi_spawn.py \
+    --world_size 8 --use_mpi run_audio_classification.py \
+    --model_name_or_path facebook/wav2vec2-base \
+    --dataset_name common_language \
+    --audio_column_name audio \
+    --label_column_name language \
+    --output_dir /tmp/wav2vec2-base-lang-id \
+    --overwrite_output_dir \
+    --remove_unused_columns False \
+    --do_train \
+    --do_eval \
+    --learning_rate 3e-4 \
+    --max_length_seconds 8 \
+    --attention_mask False \
+    --warmup_ratio 0.1 \
+    --num_train_epochs 10 \
+    --per_device_train_batch_size 16 \
+    --per_device_eval_batch_size 32 \
+    --dataloader_num_workers 8 \
+    --seed 0 \
+    --use_habana \
+    --use_lazy_mode \
+    --gaudi_config_name Habana/wav2vec2 \
+    --throughput_warmup_steps 2
+```
+
+On 8 HPUs, this script should run in ~12 minutes and yield accuracy of **80.38%**.
+
```

Comment:
seems this is not good scaling? do we know why?
should we actually put some notes as the model is not fully optimized, the optimization can be done through gaudi_config and training parameter?

Reply:
What do you mean? The multi-HPU example uses a different dataset and different hyperparameters from the single-HPU one.